color-scripts/crowns

```diff
@@ -1,11 +1,13 @@
- [1;34m    [0;30m    [5;44m [0;30m[11C[5;42;32m [0;30m[5C[37m [30m[5C[5;46;32m [0;30m[5C[37m [30m[5C[5;41;32m [0;30m[5C[37m [30m[5C[5;45;35m [0;30m[5C[37m [30m[5C[5;43;35m [0m
- [1;34m    [0;5;44;30m  [0;30m [5;44m [0;1;44;34m [0;5;44;30m [0;30m [5;44m  [0;30m   [5;42;32m  [0;30m [5;42;32m [0;42;32m [5m [0;30m [5;42;32m  [0;30m [37m [30m [5;46;32m  [0;30m [5;46;32m [0;46;32m [5m [0;30m [5;46;32m  [0;30m [37m [30m [5;41;32m  [0;30m [5;41;32m [0;41;32m [5m [0;30m [5;41;32m  [0;30m [37m [30m [5;45;35m  [0;30m [5;45;35m [0;45;35m [5m [0;30m [5;45;35m  [0;30m [37m [30m [5;43;35m  [0;30m [5;43;35m [0;43;35m [5m [0;30m [5;43;35m  [0m
- [1;34m   [5;44m [0;1;44;34m  [5m [0;1;44;34m   [5m [0;1;44;34m  [5m [0m [5;42;32m [0;42;32m  [5m [0;42;32m   [5m [0;42;32m  [5m [0m [5;46;32m [0;46;32m  [5m [0;46;32m   [5m [0;46;32m  [5m [0m [5;41;32m [0;41;32m  [5m [0;41;32m   [5m [0;41;32m  [5m [0m [5;45;35m [0;45;35m  [5m [0;45;35m   [5m [0;45;35m  [5m [0m [5;43;35m [0;43;35m  [5m [0;43;35m   [5m [0;43;35m  [5m [0m
- [1;34m   [5;44m [0;1;44;34m  [5m [0;1;44;34m   [5m [0;1;44;34m  [5m [0m [5;42;32m [0;42;32m  [5m [0;42;32m   [5m [0;42;32m  [5m [0m [5;46;32m [0;46;32m  [5m [0;46;32m   [5m [0;46;32m  [5m [0m [5;41;32m [0;41;32m  [5m [0;41;32m   [5m [0;41;32m  [5m [0m [5;45;35m [0;45;35m  [5m [0;45;35m   [5m [0;45;35m  [5m [0m [5;43;35m [0;43;35m  [5m [0;43;35m   [5m [0;43;35m  [5m [0m
-[5C[5;1;44;34m [0;1;44;34m [5m [0;1;44;34m   [5m [0;1;44;34m [5m [0m  [30m [5;42;32m [0;42;32m [5m [0;42;32m   [5m [0;42;32m [5m [0;30m [37m [30m [5;46;32m [0;46;32m [5m [0;46;32m   [5m [0;46;32m [5m [0;30m [37m [30m [5;41;32m [0;41;32m [5m [0;41;32m   [5m [0;41;32m [5m [0;30m [37m [30m [5;45;35m [0;45;35m [5m [0;45;35m   [5m [0;45;35m [5m [0;30m [37m [30m [5;43;35m [0;43;35m [5m [0;43;35m   [5m [0;43;35m [5m [0m
-[6C[5;1;44;34m       [0m   [30m  [5;42;32m       [0;30m  [37m [30m  [5;46;32m       [0;30m  [37m [30m  [5;41;32m       [0;30m  [37m [30m  [5;45;35m       [0;30m  [37m [30m  [5;43;35m       [0m
-[6C[5;1;44;34m       [0m   [30m  [5;42;32m       [0;30m  [37m [30m  [5;46;32m       [0;30m  [37m [30m  [5;41;32m       [0;30m  [37m [30m  [5;45;35m       [0;30m  [37m [30m  [5;43;35m       [0m
-    [47;35m                                                                       [0m
-    [5;47;35m                                                                       [0m
+#!/bin/bash
+
+printf " [1;34m    [0;30m    [5;44m [0;30m[11C[5;42;32m [0;30m[5C[37m [30m[5C[5;46;32m [0;30m[5C[37m [30m[5C[5;41;32m [0;30m[5C[37m [30m[5C[5;45;35m [0;30m[5C[37m [30m[5C[5;43;35m [0m
```

Comment:
This script prints the raw text for me and does not work.

Reply:
hmmm weird ... 
I'm not that good with shell scripting but I have two solutions for you that might work:

1. try to make both executable

```sh
chmod +x  ./color-scripts/crowns
chmod +x ./color-scripts.sh
```

2. change the actual script shebang from `/bin/sh` to `/bin/bash`

if it didn't work. tell me about your setup.
what shell you're using or what terminal emulator!? how do you try to do it with another terminal emulator!?

I didn't write the crowns but I tried it in these setups:

- Linux: (zsh & kitty) (zsh & xterm) (bash & kitty) (bash & xterm)
- windows (cmd with wsl bash) (cmd with git bash) (windows terminal with git bash) (windows terminal with wsl)

Reply:
```bash
acxz@archard ../acxz/shell-color-scripts/colorscripts (git)-[new-scripts] % ls -lisah crowns
10246799 4.0K -rwxr-xr-x 1 acxz acxz 2.8K Feb 20 16:41 crowns
```

It is already `#!/bin/bash`.

setup:
alacritty, zsh

Reply:
mb got it to work, I messed up copy-pasting... srry for the noise

Reply:
btw I'm adding scripts from here to the active color script repo here: https://gitlab.com/dwt1/shell-color-scripts
Looks like this repo has met its time and the color scripting community is now headquartered over there.
Just thought I'd let you know!